kuma/landing/jinja2/landing/includes/style.html

```diff
@@ -0,0 +1,8 @@
+{% stylesheet 'home_base' %}
+{% stylesheet 'callout' %}
+{% stylesheet 'home_newsletter' %}
```

Comment:
I'm wondering whether we might want to bundle these sheets into one (to minimize requests)? Or do we prefer to have them separate so that changes to one in the future don't cause cache misses in the other?

Reply:
We could, but we wanted to experiment with many(up to a sane limit 😸) bundles to see how it impacts performance via HTTP2. As the landing page is not the most highly prized property on MDN I reckon this might be a could place to test this.

Reply:
Hey Schalk, do you have docs for your test? What are you using to measure it? I didn't realize we were doing that, maybe I forgot.

Reply:
Hey @atopal, not docs so much but there is this issue:
https://github.com/mdn/kuma/issues/5867

The basic premise being, that we serve one large CSS bundle and thus, we are not playing to the strengths of HTTP2. In order to test out our hypothesis, we started to break up the large single bundle into smaller bundles. I have here taken a larger stab at it in order to determine whether there are any gains, and if so, how much.

The other side of this is to also move towards inlining critical CSS which depends a lot on having these smaller focused bundles.

Reply:
Okay, how are we determining whether there are any gains?

Reply:
We do have monitoring via Speedcurve for the landing page correct? If not, then I would recommend we set something up to test TFP and TTI. Thoughts?

Reply:
I'm on-board with testing First Contentful Paint and First Input Delay (TFP is not that meaningful, and [TTI is a more of a lab measure not for real user metrics](https://developers.google.com/web/updates/2018/05/first-input-delay)). However, it needs to be designed properly, so we can actually tell whether there was a benefit or not. I don't have the time to do that right now, because it's not a high priority, we currently don't have major performance issues, especially on the start page, but if you want to go for that feel free.

Reply:
I suggest we move any efforts of doing that kind of empirical perf testing till we're in Yari. Even if that means waiting many many months till we have the infra to start such tests. 
Then you'd be able to spin up a test version and bombard it with Lighthouse a bunch of times and see if there's a difference. 

For now, let's not worry. A subtle advantage with making it many distinct bundles the possible chance of browser cache re-use.

Reply:
I get what you are saying. @Gregoor, do you foresee a downside to merging this as is?

Reply:
If we have a way to measure it, I'm happy. Don't see any other downsides.

Reply:
Speedcurve is currently tracking the performance of the landing page so, after a couple of days, we should be able to have some data.